modules/psap-driver-toolkit-using.adoc

```diff
@@ -120,6 +120,7 @@ $ sed "s#DRIVER_TOOLKIT_IMAGE#${DRIVER_TOOLKIT_IMAGE}#" 0000-buildconfig.yaml.te
 ====
 The driver toolkit was introduced to {product-title} 4.6 as of version 4.6.30, in 4.7 as of version 4.7.11, and in 4.8.
 ====
+//Is this note still required?
```

Comment:
@dagrayvid, will you please confirm that this note is still necessary for OCP 4.9?

Reply:
Can probably be removed.